core/src/main/java/io/questdb/std/datetime/microtime/Micros.java

```diff
@@ -42,18 +42,20 @@
 public final class Micros {
     public static final long DAY_MICROS = 86_400_000_000L; // 24 * 60 * 60 * 1000 * 1000L
     public static final long AVG_YEAR_MICROS = (long) (365.2425 * DAY_MICROS);
-    public static final long DAY_SECONDS = 86400;
+    public static final long DAY_SECONDS = 86400L;
     public static final long FIRST_CENTURY_MICROS = -62135596800000000L;
     public static final long HOUR_MICROS = 3600000000L;
-    public static final long HOUR_SECONDS = 3600;
-    public static final long MICRO_NANOS = 1000;
-    public static final long MILLI_MICROS = 1000;
-    public static final long MINUTE_MICROS = 60000000;
-    public static final long MINUTE_SECONDS = 60;
+    public static final long HOUR_MILLIS = 3600000L;
```

Comment:
this constant is misplaced, there is Millis class for tooling used by MillisDriver. Please move

Reply:
There is a `Dates` class though inside the `millis` package which seems to have few constants like `HOUR_MILLIS`, `MINUTE_MILLIS`, `SECOND_MILLIS`.
It is a bit messy, but I can add my stuff to `Dates` then.